docs/index.asciidoc

```diff
@@ -8,6 +8,6 @@ include::overview.asciidoc[]
 
 include::installation.asciidoc[]
 
-
+include::compatibility.asciidoc[]
```

Comment:
If you move the compatibility info under `Installation`, then this item can be deleted.
```suggestion
```